bot/exts/recruitment/talentpool/_review.py

```diff
@@ -433,11 +435,28 @@ async def _previous_nominations_review(self, member: Member) -> Optional[str]:
 
         nomination_times = f"{num_entries} times" if num_entries > 1 else "once"
         rejection_times = f"{len(history)} times" if len(history) > 1 else "once"
+        thread_jump_urls = []
+
+        for nomination in history:
+            try:
+                thread = await get_or_fetch_channel(nomination.thread_id)
```

Comment:
Should check for it being none first
```suggestion

        for nomination in history:
            if nomination.thread_id is None:
                continue
            try:
                thread = await get_or_fetch_channel(nomination.thread_id)
```